src/Microsoft.AspNet.Mvc.Razor.Host/Directives/ChunkInheritanceUtility.cs

```diff
@@ -13,7 +13,7 @@
 namespace Microsoft.AspNet.Mvc.Razor.Directives
 {
     /// <summary>
-    /// A utility type for supporting inheritance of tag helpers and chunks into a page from applicable _ViewStart
+    /// A utility type for supporting inheritance of tag helpers and chunks into a page from applicable _GlobalImport
```

Comment:
know you didn't touch this part but it jumped out at me: `tag helpers` => `<see cref="ITagHelper" />s`, and `chunks` => `<see cref="Chunk" />s`